test/tools/ossfuzz/SolidityMutator.cpp

```diff
@@ -0,0 +1,125 @@
+/*
+	This file is part of solidity.
+
+	solidity is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	solidity is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with solidity.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include <test/tools/ossfuzz/SolidityMutator.h>
+
+#include <libsolutil/Whiskers.h>
+
+#include <boost/algorithm/string/join.hpp>
+
+#include <algorithm>
+
+using namespace std;
+using namespace solidity::util;
+using namespace solidity::test::fuzzer;
+
+string SolidityMutator::genRandString(unsigned int _length, char _start, char _end)
+{
```

Comment:
It won't work properly unless `_start` <= `_end`, right?
```suggestion
string SolidityMutator::genRandString(unsigned int _length, char _start, char _end)
{
	assert(_start <= _end);
```

I think it's really easy to make that mistake when it comes to symbols and it would be useful to be able to detect such mistakes.